xwiki-platform-core/xwiki-platform-netflux/xwiki-platform-netflux-api/src/main/resources/ApplicationResources_de.properties

```diff
@@ -17,6 +17,38 @@
 # Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 # 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 # ---------------------------------------------------------------------------
+admin.edit.realtime=Echtzeit-Editoren
+admin.edit.realtime.description=Konfiguriere die Echtzeit-Editoren.
+XWiki.Realtime.ConfigurationClass_useGlobalConfig=Globale Konfiguration
+XWiki.Realtime.ConfigurationClass_useGlobalConfig.hint=Verwende die Konfiguration des Hauptwikis
+XWiki.Realtime.ConfigurationClass_toolbarUserlist=Verbundene Nutzer anzeigen
+XWiki.Realtime.ConfigurationClass_toolbarUserlist.hint=Die Liste der verbundenen Benutzer oben im Bearbeitungsbereich anzeigen
+XWiki.Realtime.ConfigurationClass_toolbarUserlist_name=Nur Usernamen anzeigen
+XWiki.Realtime.ConfigurationClass_toolbarUserlist_avatar=Nur Useravatare anzeigen
+XWiki.Realtime.ConfigurationClass_toolbarUserlist_both=Usernamen und -avatare anzeigen
+XWiki.Realtime.ConfigurationClass_marginAvatar=Bearbeitende Benutzer anzeigen
+XWiki.Realtime.ConfigurationClass_marginAvatar.hint=Zeige an der Seite des Bearbeitungsbereichs die Avatare der Benutzer an, die gerade tippen
+XWiki.Realtime.ConfigurationClass_enableMerge=Zusammenf\u00FChren beim Speichern
+XWiki.Realtime.ConfigurationClass_enableMerge.hint=Versuche, den Inhalt beim Speichern eines Dokuments zusammenzuf\u00FChren (nur deaktivieren, wenn dies Probleme verursacht)
+# ---------------------------------------------------------------------------
+# See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
```

Comment:
This is a surprising change... We need to understand why it's like that.

Reply:
This is a classic actually: it comes from the fact that `realtime.editingAlone` used to be the first key (to which the comment above was associated as a key comment).

Reply:
> This is a classic actually: it comes from the fact that realtime.editingAlone used to be the first key (to which the comment above was associated as a key comment).

Interesting, so the first key has been moved by adding new keys in the file above it. But then it means the comment on this key should have been removed in weblate: the source file should have been parsed again. 

> Ideally, the header should be skipped when parsing the file and regenerated when generating the file.

Yeah AFAIR it wasn't easy to do that in a generic way.

Reply:
>  the source file should have been parsed again.

It seems it does eventually (there is no comment on https://l10n.xwiki.org/translate/xwiki-platform/xwiki-platform-netflux-api/en/?checksum=20fe1ac3f204157c&sort_by=-priority,position#nearby) but there must be some race condition at some point. What is sure is that we always add this issue when a first key moved down.

Reply:
ok, thanks for the hint.

Reply:
Maybe Weblate has different comments from the main key and the translation and since this commit is the one in which the key was moved down from de translation point of view it might have the comment of the previous version of the key and the serializer generating a new header.